### PR TITLE
ICMSLST-2911 Refresh dev template data

### PR DESCRIPTION
**Pull request by mikeylockwood**

### CODE REVIEW

File: data_migration/management/commands/post_migration.py [hunk 1]
```diff
@@ -53,9 +50,10 @@ def add_data_to_v2_additional_models(self):
         add_region_to_existing_countries(self.stdout)
         add_inactive_countries()
         add_country_translations()
-        add_user_management_email_templates()
         call_command("set_icms_sites")
 
+        # TODO: Run update update_email_data command? Command also creates 2 necessary email templates
```
Comment (MattHolmes123) [inline data_migration/management/commands/post_migration.py:55]: Jira ticket or it didn't happen :)

File: web/domains/template/views.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ class TemplateListView(ModelFilterView):
     filterset_class = TemplatesFilter
     page_title = "Maintain Templates"
     permission_required = Perms.sys.ilb_admin
-    ordering = ("template_type", "template_name")
+    ordering = ("-is_active", "template_type", "template_name")
```
Comment (MattHolmes123) [inline web/domains/template/views.py:37]: Does this order by active or inactive at the top?

**Review by MattHolmes123 — APPROVED** [review]

File: data_migration/management/commands/post_migration.py (hunk 1 above)
Comment (marcuspp) [reply data_migration/management/commands/post_migration.py:55]: I have a ticket to sort this, that i am working on next

**Review by marcuspp — APPROVED** [review]

File: web/domains/template/views.py (hunk 2 above)
Comment (mikeylockwood) [reply web/domains/template/views.py:37]: Active at the top, 1 then 0 instead of 0 then 1

**Pull request merged**